http/pkger_http_server_test.go

```diff
@@ -261,75 +263,57 @@ func TestPkgerHTTPServer(t *testing.T) {
 	})
 }
 
-func bucketPkg(t *testing.T, encoding pkger.Encoding) *pkger.Pkg {
+func bucketPkgKinds(t *testing.T, encoding pkger.Encoding) []pkger.Kind {
 	t.Helper()
 
 	var pkgStr string
 	switch encoding {
 	case pkger.EncodingJsonnet:
 		pkgStr = `
 local Bucket(name, desc) = {
+    apiVersion: 'influxdata.com/v2alpha1',
```

Comment:
can you make a global constant for the api version, also i think we should use beta1, instead of alpha1